KenticoCloud.Delivery/DeliveryClient.cs

```diff
@@ -18,6 +19,12 @@ public sealed class DeliveryClient : IDeliveryClient
         private readonly DeliveryEndpointUrlBuilder _urlBuilder;
 
         private IContentLinkUrlResolver _linkUrlResolver;
+
+        /// <summary>
+        /// Richtext content items processor for content items retrieved using this client.
+        /// </summary>
+        public InlineContentItemsProcessor InlineContentItemsProcessor { get; private set; }
```

Comment:
Not sure what you mean as it isn't clear from issue description. Probably would be better if you did it yourself.